web/gui/dashboard_info.js

```diff
@@ -267,6 +267,12 @@ netdataDashboard.menu = {
         info: 'Performance metrics for <b>RetroShare</b>. RetroShare is open source software for encrypted filesharing, serverless email, instant messaging, online chat, and BBS, based on a friend-to-friend network built on GNU Privacy Guard (GPG).'
     },
 
+    'riak': {
+        title: 'Riak',
```

Comment:
Riak KV

Reply:
> title

Reply:
Changed in 71baea7939079c01b75878ac094681a5c093e16d.